src/plugins/ui_core_views/evaluation_conditional_format.ts

```diff
@@ -320,6 +320,11 @@ export class EvaluationConditionalFormatPlugin extends UIPlugin {
   ) {
     const targetCol = col - zone.left + targetZone.left;
     const targetRow = row - zone.top + targetZone.top;
+    // if the target cell is outside the CF rangeValues
+    // we return -1 to avoid applying the data bar rule
```

Comment:
I don't like the idea of a special case of data bar rule in a method that seems generic and supposed to return an evaluated cell.
I'd prefer if we inline this method where it's used